src/ApprenticeAan/SFA.DAS.ApprenticeAan.Api/Models/GetCalendarEventsModel.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ApprenticeAan.Application.CalendarEvents.Queries.GetCalendarEvents;
+using SFA.DAS.ApprenticeAan.Application.Common;
+using SFA.DAS.ApprenticeAan.Application.Infrastructure.Configuration;
+
+namespace SFA.DAS.ApprenticeAan.Api.Models;
+
+public class GetCalendarEventsModel
```

Comment:
Suggest rename to `GetCalendatEventsRequestModel` to differentiate between response and request models